### ISSUE TITLE
ISO_3166-1_alpha-2 CW (Curaçao) is missing

### ISSUE THREAD

**Issue by jorbie** [sshaw/normalize_country#4]
thx for your great work so far!
can you plz add Curaçao to your list?

```
CW:
  aliases:
  - Curacao
  alpha2: CW
  alpha3: CUW
  fifa: CUW
  ioc:
  iso_name: Curaçao
  numeric: "533"
  official: Curaçao
  short: Curaçao
emoji:

```

**Comment by sshaw** [comment]
Great thanks, always wanted to go here. Added to master. 

Note that their numeric ISO code is 531.

**Comment by sshaw** [comment]
Will do a release tonight.

**Comment by sshaw** [comment]
Included in v0.2.2. 

Using the current convention Curaçao has no short so this has been removed.

**Comment by jorbie** [comment]
works nice now :) - thx for your quick release!
